src/compositions/question-form-10.tsx

```diff
@@ -0,0 +1,43 @@
+import { FC } from 'react';
+import { InputField } from '../elements/input-field';
+import { useRouter } from 'next/dist/client/router';
+
+type Props = {
+  label?: string;
+  initialAnswer?: string;
+  href: string;
+  answer: { [key: string]: string };
+  callback: React.Dispatch<React.SetStateAction<{ [key: string]: string }>>;
+};
+
+const validateInput = (answer: string) => {
+  if (parseFloat(answer) > 10000) {
+    return 'Bist du sicher? das sind sehr viele km.';
+  } else if (answer.length > 10) {
+    return 'Bitte gib maximal 10 Ziffern nach dem Komma ein.';
+  }
+  return null;
+};
+
+export const Question10: FC<Props> = ({ label, initialAnswer, href, answer, callback }) => {
+  const router = useRouter();
+
+  return (
+    <div>
+      <InputField
+        type="number"
+        label={label || 'km'}
+        name="answer"
+        id="number"
+        step="10"
+        placeholder={initialAnswer || '100'}
```

Comment:
If there _is_ an `initialAnswer` should you pass that as a `value` rather than a `placeholder`?